src/test/java/team7/hrbank/unit/department/RepositoryTest.java

```diff
@@ -0,0 +1,222 @@
+        Integer idAfter = 0;  // 커서 (현재 가져온 페이지의 마지막 id)
+        String cursor = null; // 커서 (이전 페이지 마지막 커서)
+        Integer requestSize = 10;
+        String sortedField = "establishmentDate"; // 정렬 필드(name or establishmentDate)
+        String sortDirection = "asc ";
+        DepartmentSearchCondition searchCondition = DepartmentSearchCondition.builder()
+                .nameOrDescription(nameOrDescription)
+                .idAfter(idAfter)
+                .cursor(cursor)
+                .size(requestSize)
+                .sortedField(sortedField)
+                .sortDirection(sortDirection)
+                .build();
+
+        // 저장될 엔티티 수
+        int entitySettingSize = 12;
+        int repeatCount = 5;
+        String otherNameOrDescriptionSize = "기서";
+        for (int i = 0; i < repeatCount; i++) {
+            setting_entity_save_and_containing_name(entitySettingSize, searchCondition.getNameOrDescription());
+            setting_entity_save_and_containing_name(entitySettingSize, otherNameOrDescriptionSize);
+        }
+
+        //when
+        List<DepartmentPageContentDTO> contentDTOList = new ArrayList<>();
+        DepartmentResponseDTO result;
+        do {
+            result = departmentRepository.findPagingAll1(searchCondition);
+            contentDTOList.addAll(result.contents());
+
+            // 조건 다시 세팅
+            searchCondition = DepartmentSearchCondition.builder()
+                    .nameOrDescription(nameOrDescription)
+                    .idAfter(result.nextIdAfter())
+                    .cursor(result.nextCursor())
+                    .size(requestSize)
+                    .sortedField(sortedField)
+                    .sortDirection(sortDirection)
+                    .build();
+        } while (result.hasNext());
+
+        // then 1. 기본 값 테스트
+        assertThat(contentDTOList).as("필터를 거치고 실제로 가져온 DTO가 예상과 같은지")
+                .hasSize(entitySettingSize * repeatCount)
+                .as("전부 필터링 조건에 맞는지")
+                .allSatisfy(department -> assertThat(department.name()).contains(nameOrDescription));
+
+        // then 2. 정렬 테스트
+        if (sortDirection.trim().equalsIgnoreCase("desc")) {
+            assertThat(contentDTOList).as("Established 내림차순 정렬 확인 후 같을 경우 -> 2번 째 정렬 기준인 id를 내림차순 정렬 확인")
+                    .isSortedAccordingTo(
+                            Comparator.comparing(DepartmentPageContentDTO::establishmentDate, Comparator.reverseOrder())
+                                    .thenComparing(DepartmentPageContentDTO::id, Comparator.reverseOrder())
+                    );
+
+        } else {
+            assertThat(contentDTOList).as("Established 오름차순 정렬 확인 후 같을 경우 -> 2번 째 정렬 기준인 id를 오름차순 정렬 확인")
+                    .isSortedAccordingTo(
+                            Comparator.comparing(DepartmentPageContentDTO::establishmentDate, Comparator.naturalOrder())
+                                    .thenComparing(DepartmentPageContentDTO::id));
+        }
```

Comment:
`String sortDirection = "asc " `이니  if-else문에서 else문만 타는 상황인데요,  
과감히 if문 제거하는게 좋을 것 같아요.

테스트코드는 개발자들이 읽는 문서이기때문에 가독성에 더 신경써야 합니다.  


```suggestion
        // then 2. 정렬 테스트
            assertThat(contentDTOList).as("Established 오름차순 정렬 확인 후 같을 경우 -> 2번 째 정렬 기준인 id를 오름차순 정렬 확인")
                    .isSortedAccordingTo(
                            Comparator.comparing(DepartmentPageContentDTO::establishmentDate, Comparator.naturalOrder())
                                    .thenComparing(DepartmentPageContentDTO::id));
        }
```